src/main/java/org/terasology/climateConditions/BodyTemperatureChangedEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.climateConditions;
+
+import org.terasology.entitySystem.event.Event;
+
+public class BodyTemperatureChangedEvent implements Event {
```

Comment:
I'd like to put some emphasis on providing **two events** for the body temperature system - one for the numeric change and one translating the numeric change into the abstract category. 

While you may just want to use the categories, exposing the numeric underlying numeric change may be crucial for other modules (to come). More importantly, separating the logic by these events prepares us to easily allow for extensions here.

Reply:
The numeric value would keep on changing every second, do we want to send out an event every second signifying change in numeric values?
Also AffectBodyTemperatureEvent will be sent out every second anyway(although fAacing some issues with that right now), couldn't the other systems just utilize that when need arises?